src/scale.ts

```diff
@@ -136,20 +136,67 @@ export interface ScaleConfig {
   /** Default range for shape */
   shapeRange?: string[];
 
-  /** Default range for bar size scale */
-  barSizeRange?: number[];
+  /**
+   * Default minimum range for bar size scale with zero=false.
+   * @minimum 1
```

Comment:
In this case, users set zero=false thus they don't expect domain to include zero.  
(Otherwise, they wouldn't bother to override the zero value.)

Thus, if you want to set a theme for such cases, I don't think including setting rangeMin in a config is a good idea. 

In any case, users can explicitly set size's scale range to start from zero. 

That said, if you still prefer that we don't include `@minimum 1`, I'm happy to change.